tests/Resource/CloudAccount/resources/GET-%2Fcloud-account%2F1%2Fbackup.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "filepath": "/home/a2094f12/mag.example.com/iworx-backup/filename.tgz",
+    "filename": "filename.tgz",
+    "filesize": "123 MB",
+    "filesize_bytes": 456,
+    "type": "Full Backup",
+    "domain_options": "single-domain",
+    "filedate": 1541094489,
+    "complete": true,
+    "download_url": "https://69.160.55.105:2443/siteworx/index?action=sso&sid=30cdb01ab2be54d60d52a3232970f288&uri=L3NpdGV3b3J4L2Rvd25sb2FkP2FjdGlvbj1iYWNrdXAmZmlsZT1tYWcuZXhhbXBsZS5jb20lMkJmdWxsLXNpbmdsZS1Ob3YuMDEuMjAxOC0xMy40Ny40MC50Z3omZG9tYWluPW1hZy5leGFtcGxlLmNvbQ%3D%3D"
```

Comment:
RFC 5737 reserves `203.0.113.0/24` for examples; let's pick one from there.

Also, is this a real siteworx sso token? need to obfuscate (or simply omit; it's not needed for the test).

same comments below.